packages/http-client-java/generator/http-client-generator-test/pom.xml

```diff
@@ -37,23 +37,23 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-test</artifactId>
-        <version>1.26.2</version>
+        <version>1.27.0-beta.1</version>
```

Comment:
Should we use beta version here?

Reply:
Alan made this lib always beta from now on (as this is lib only used in test, so they don't want take the obligation that the API must be stable).